src/ServiceControl.Audit/Infrastructure/PublishFullTypeNameOnlyBehavior.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Particular.ServiceControl
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Pipeline;
+
+    class PublishFullTypeNameOnlyBehavior : Behavior<IOutgoingPhysicalMessageContext>
```

Comment:
FYI This is needed for any integration events to work so perhaps I should make this a separate PR?